main.go

```diff
@@ -38,9 +36,21 @@ func main() {
 }
 
 func _main() int {
-	defer log.Flush()
-	logger.SetupLogger(logger.GetLogFileLocation(defaultLogFilePath))
-
+	/*	config := log.Configuration{
+			EnableConsole:     false,
+			ConsoleLevel:      "Debug",
+			ConsoleJSONFormat: true,
+			EnableFile:        true,
+			FileLevel:         "Debug",
+			FileJSONFormat:    true,
+			FileLocation:      defaultLogFilePath,
+		}
+		err := log.NewLogger(config)
+		if err != nil {
+			fmt.Printf("Could not instantiate log %s", err.Error())
+			os.Exit(1)
+		}
+	*/
```

Comment:
Please remove the above comment. I think it's leftover from debugging? Alternately, make it actual code :)